lib/data_handlers/json_handler.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+#  class for loading json data
+class JsonHandler
+  DEFAULT_JSON_FILE_LOCATION = 'data/people.json'
+
+  def get_person_dtos(sort)
+    person_dtos = []
+    data = load_json
+    data.each do |person|
+      person_dto = PersonDto.new
```

Comment:
I would make the `PersonDto` class take named arguments for each attribute within the `initialize` method to allow this to be written a little cleaner e.g.

`PersonDto.new(id: person['id'], first_name: person['first_name'])`